install/upgrade/versions/1.5.8.sh

```diff
@@ -30,4 +30,14 @@ for user in $(grep "$HOMEDIR" /etc/passwd |egrep "$shells" |cut -f 1 -d:); do
     if [ -d "$HESTIA/data/users/$user" ]; then
     $BIN/v-add-user-sftp-jail "$user" "no"
     fi
-done
\ No newline at end of file
+done
+
+echo "[ * ] Add .bash_history for existing bash shell users"
+for user in $($BIN/v-list-users plain | cut -f1); do
+    for shell in $($BIN/v-list-user $user plain | cut -f20); do
```

Comment:
I'm not a fan of cut -f20, it may fail when we change v-list-user. Maybe someone has a better idea how we can grab the user's shell.

Reply:
We got get shell value from /etc/password like we do sftp jail